Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -484,15 +484,15 @@ void Settings::SetBatchModeEnabled(bool batch)
 #ifdef USE_DISCORD_PRESENCE
```

Comment:
This should be moved to UICommon.

Currently, you have (almost) the same exact function in DolphinWX.